src/engine/BMSkillBerserk.php

```diff
@@ -91,6 +125,11 @@ protected static function get_interaction_descriptions() {
         );
     }
 
+    /**
+     * Does this skill prevent the determination of whether a player can win?
```

Comment:
So... to me, comments like this one don't seem very useful.  I get that copy/paste is the easiest way to shut up phpdoc, but maybe that means we simply shouldn't run phpdoc?  In particular, this isn't useful because:
* A relevant piece of metadata about this function is that it overrides the base function in BMSkill (that you actually *could* say and copy/paste among all the children, so it doesn't require a lot more work)
* You're commenting a function in which BMSkillBerserk overrides the base `prevents_win_determination()` value, and the code of the function shows that the Berserk skill does prevent win determination, but the new comment doesn't say a word about *why*.

My takeaway is, if we're not going to use comments to explain design decisions, why write comments?  And if we're not going to use the phpdoc check as an excuse to write comments that explain design decisions, i'd rather disable the phpdoc check.

In particular, obviously, `BMSkillBerserk::prevents_win_determination()` is a simple function and it's pretty easy for anyone with a passing familiarity with buttonmen to figure out why it might return true for berserk dice.  But if i were commenting this, i'd say something more like:
```php
/**
 * Override the base function from BMSkill because dice change size when they participate
 * in Berserk attacks, which prevents determination of whether a player can still win the round.
```

Or, y'know, whatever.  Anyway, i can also just blindly approve this pull on the theory that it isn't very important, and i haven't read down very far, so there may be more informative comments below, or it may be that having the `@return` statements is helpful enough that we should keep the phpdoc check to force us to write those and just intentionally not worry about the function description strings.

But, basically, i'd like your two cents on what the purpose of adding these comments is.